modules/apps/commerce/commerce-wish-list-service/src/main/java/com/liferay/commerce/wish/list/service/impl/CommerceWishListItemLocalServiceImpl.java

```diff
@@ -187,15 +212,36 @@ protected void activate(Map<String, Object> properties) {
 			CommerceWishListConfiguration.class, properties);
 	}
 
+	private CommerceWishListItem _updateCommerceWishListItem(
+			long commerceWishListId, long cProductId, String cpInstanceUuid,
```

Comment:
I meant a new service method not a private one.. With a service method we could fix the add to wishlist with options..